scoap3/articles/api/views.py

```diff
@@ -28,6 +29,7 @@ class ArticleViewSet(
 ):
     queryset = Article.objects.all()
     serializer_class = ArticleSerializer
+    permission_classes = [AllowAnyGetAuthenticatedOthers]
```

Comment:
you can use `IsAuthenticatedOrReadOnly` from https://www.django-rest-framework.org/api-guide/permissions/#isauthenticatedorreadonly